Tutorial/GettingStartedGuide.md

```diff
@@ -0,0 +1,148 @@
+# Getting Started Guide  
+  
+This guide explains how to package up your trained model as a web service with a predict API that can be scanned using Certifai
+  
+  
+## Table of Contents  
+1. [pre-requisites](#pre-requisites)   
+2. [predict](#predict)  
+3. [api](#api)  
+4. [build-container](#build-container)  
+5. [run-container](#run-container)  
+5. [test-container](#test-container)  
+  
+## pre-requisites  
+  
+1. trained model
+2. [source-to-image (s2i)](https://github.com/openshift/source-to-image)  
+3. [Httpie](https://httpie.org/) (Required to run `make test`)   
+  
+## predict     
+ Create a predict function that takes a message object as function parameter containing data instances to predict against   
+and return list of predictions   
+
+   > refer to function `predict_titanic_survivor` in `titanic_survivor_prediction/predict.py`
+  
+##  api
+  
+Exposing model predict as an api using [s2i](https://github.com/openshift/source-to-image) requires  
+  
+1. creating a `.s2i/environment` config file in the top-level directory with  
+   * `ROUTES=<routeName:predictModuleName:predictFunctionName>`  
+
+  > e.g. `ROUTES=titanic_survivor_rf/predict:predict:predict_titanic_survivor` 
+
+2 . creating a `requirements.txt` to install python-dependencies required to run the model inside the container
+
+
+## build-container 
+
+This command builds the container and packages the code for the model predict web api  
+
+1. s2i build . builderImageName containerName 
+   * builderImage (public): `cortex-s2i-daemon-python36-slim:1.0.0`  
+ * containerName: name to identify container by 
+  
+e.g. 
```

Comment:
which folder should user be in when they run this command?

Reply:
makes sense. there is a make file with the build,run and test commands though